Merge remote-tracking branch 'origin/master'

Conflicts:
	CoreConceptsPy/fields.py
	CoreConceptsPy/test/fields_test.py
#
# It looks like you may be committing a merge.
# If this is not correct, please remove the file
#	.git/MERGE_HEAD
# and try again.


# Please enter the commit message for your changes. Lines starting
# with '#' will be ignored, and an empty message aborts the commit.
# On branch master
# Your branch and 'origin/master' have diverged,
# and have 1 and 15 different commits each, respectively.
#   (use "git pull" to merge the remote branch into yours)
#
# All conflicts fixed but you are still merging.
#
# Changes to be committed:
#	modified:   .gitignore
#	new file:   ConceptsOfSpatialInformation.sublime-project
#	modified:   CoreConceptsPy/.gitignore
#	modified:   CoreConceptsPy/Makefile
#	new file:   CoreConceptsPy/__init__.py
#	renamed:    CoreConceptsPy/coreconcepts_oo.py -> CoreConceptsPy/coreconcepts.py
#	modified:   CoreConceptsPy/data/fields/testFocal.tif
#	new file:   CoreConceptsPy/data/fields/zone.tif
#	new file:   CoreConceptsPy/events.py
#	deleted:    CoreConceptsPy/events_impl.py
#	new file:   CoreConceptsPy/examples/__init__.py
#	new file:   CoreConceptsPy/examples/events/__init__.py
#	new file:   CoreConceptsPy/examples/events/example-1.py
#	new file:   CoreConceptsPy/examples/fields/__init__.py
#	new file:   CoreConceptsPy/examples/locations/__init__.py
#	new file:   CoreConceptsPy/examples/networks/__init__.py
#	renamed:    CoreConceptsPy/data/networks/README.md -> CoreConceptsPy/examples/networks/data/README.md
#	renamed:    CoreConceptsPy/data/networks/USAir97.net -> CoreConceptsPy/examples/networks/data/USAir97.net
#	renamed:    CoreConceptsPy/data/networks/karate.gml -> CoreConceptsPy/examples/networks/data/karate.gml
#	renamed:    CoreConceptsPy/data/networks/power.gml -> CoreConceptsPy/examples/networks/data/power.gml
#	new file:   CoreConceptsPy/examples/networks/karate.py
#	new file:   CoreConceptsPy/examples/objects/__init__.py
#	renamed:    CoreConceptsPy/fields_oo.py -> CoreConceptsPy/fields.py
#	deleted:    CoreConceptsPy/fields_impl.py
#	renamed:    CoreConceptsPy/locate_impl.py -> CoreConceptsPy/locations.py
#	renamed:    CoreConceptsPy/networks_impl.py -> CoreConceptsPy/networks.py
#	deleted:    CoreConceptsPy/networks_test.py
#	renamed:    CoreConceptsPy/objects_oo.py -> CoreConceptsPy/objects.py
#	deleted:    CoreConceptsPy/objects_impl.py
#	new file:   CoreConceptsPy/test/README.md
#	new file:   CoreConceptsPy/test/__init__.py
#	renamed:    CoreConceptsPy/events_test.py -> CoreConceptsPy/test/events_test.py
#	renamed:    CoreConceptsPy/fields_test.py -> CoreConceptsPy/test/fields_test.py
#	new file:   CoreConceptsPy/test/networks_test.py
#	renamed:    CoreConceptsPy/objects_test.py -> CoreConceptsPy/test/objects_test.py
#	deleted:    CoreConceptsPy/tests.py
#
# Untracked files:
#	CoreConceptsPy/data/fields/zone.tif.vat.dbf.ALBATROSS.39248.64.sr.lock
#

